content/documentation/getting-started/index.adoc

```diff
@@ -69,36 +69,40 @@ If you prefer to use the latest Kiali version then do not install Kiali during t
 
 === Install Kiali Operator
 
-If you did not install Kiali with Istio or Maistra, or want to replace the installed version of Kiali, it is done using the Kiali Operator. The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
+If you did not install Kiali with Istio or Maistra, or want to replace the installed version of Kiali, it is done using the Kiali Operator. The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. The Kiali Operator watches the Kiali CR (Custom Resource, yaml that holds the Kiali configuration).  When the Kiali CR changes the operator installs, uninstalls, or updates Kiali as needed.
```

Comment:
That CR is not going to mysteriously change itself is it? Could we revise this to read "When you modify the Kiali CR, the operator installs, updates, or uninstalls Kiali as needed."

Reply:
+1